packages/components/src/theme/ComposerTheme/Colors/colors.ts

```diff
@@ -56,6 +56,8 @@ export const defaultColors = {
   'background-warning': '#FFE5C1',
   'background-success': '#DEFFEC',
   'background-inform': '#E4F2FF',
+  'background-skeleton-start': '#E2E8F0',
+  'background-skeleton-end': '#EDF2F7',
```

Comment:
I know this is a design decision but IMO the difference between this two values is _too subtle_ to be appreciated